src/databricks/labs/ucx/install.py

```diff
@@ -29,7 +29,8 @@
 TAG_STEP = "step"
 TAG_APP = "App"
 NUM_USER_ATTEMPTS = 10  # number of attempts user gets at answering a question
-
+EXTRA_TASK_PARAMS = {"job_id": "{{job_id}}", "run_id": "{{run_id}}", "parent_run_id": "{{parent_run_id}}",
+                     "task_key": "{{task_key}}"}
```

Comment:
Why do we need task key if we already send task name?

Reply:
Just in case

Reply:
Removed